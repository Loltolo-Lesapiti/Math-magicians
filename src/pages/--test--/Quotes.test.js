import React from 'react';
import TestRenderer from 'react-test-renderer';
import Quote from '../Quotes';
import '@testing-library/jest-dom';

describe('Quote Page', () => {
  test('Testing Quote page ', () => {
    const tree = TestRenderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
