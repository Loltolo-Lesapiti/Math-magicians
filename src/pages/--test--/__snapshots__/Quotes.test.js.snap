// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Page Testing Quote page  1`] = `
<div
  className="QuoteContainer"
>
  <h2
    className="quote"
  >
    Funny Maths Quotes
  </h2>
  <ul>
    <li>
      <p>
        If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
      </p>
      <h2>
        — John von Neumann, American-hungarian mathematician
      </h2>
    </li>
    <li>
      <p>
        If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
      </p>
      <h2>
        — John von Neumann, American-hungarian mathematician
      </h2>
    </li>
    <li>
      <p>
        If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is.
      </p>
      <h2>
        — John von Neumann, American-hungarian mathematician
      </h2>
    </li>
  </ul>
</div>
`;
